package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "aurelia-tools-ts",
+    "version": "0.1.0",
+    "description": "The shared set of tools that are used for Aurelia's TypeScript builds.",
+    "keywords": [
+      "aurelia",
+      "tools",
+      "typescript"
+    ],
+    "homepage": "http://aurelia.io",
+    "bugs": {
+      "url": "https://github.com/aurelia/tools-ts/issues"
+    },
+    "license": "MIT",
+    "author": "aurelia",
+    "repository": {
+      "type": "git",
+      "url": "http://github.com/aurelia/tools-ts"
+    }
+  }
```

Comment:
Please add `files`, see aurelia/framework#735
This repo can be a good template for all others.

Reply:
@jods4 Add it with what, empty array?  Can't we fill it as we add content?

Reply:
Usually `files: ['dist']` but you got me confused. 

What in this project is going to be used by projects referencing it? Just the base `tsconfig.json`?
In that case: `files: ['tsconfig.json']`.

Anyway, if we don't know what to put in there, I think it's still a good example to start with an empty array and fill it later.

Note that standard stuff such as `README.md`, `LICENSE`, `package.json` & co. are automatically included, no need to put them in `files`.

Reply:
@jods4 For now the `tsconfig.json`s. After we get them sorted, a `tslint.json` would be nice. Probably the scripts that are used when building the docs should also go here.
So I'll set it to `[ ]` for now.